ui/js/controllers/logviewer.js

```diff
@@ -376,6 +401,21 @@ logViewerApp.controller('LogviewerCtrl', [
             });
         }
 
+        function displayErrorLines(errors) {
+            var css = errors
+              .reduce((line, err) => `${line}a[id="${err.line_number + 1}"]+span, `, '')
```

Comment:
@eliperelman would it be smart to remove all the spaces to make the string smaller? No spaces between the `+` sign and after the comma `,`

```javascript
var css = errors
  .map(({ line_number }) => `a[id="${line_number + 1}"]+span`)
  .join(',')
```